README.md

```diff
@@ -12,7 +12,7 @@ CLI="ethereum-ics20-cli"
 ## erc20
 ```bash
 ${CLI} erc20 balance --rpc-address=${rpc_address} --wallet-address=${wallet_address} --denom=${denom}
-${CLI} erc20 transfer --rpc-address=${RPC_ADDRESS} --mnemonic="${MNEMONIC}" --ics20-bank-address=${ICS20_BANK_ADDRESS} --ics20-transfer-bank-address=${ICS20_TRANSFER_BANK_ADDRESS} --from-index=${from_index} --to-address=${to_address} --amount=${amount} --denom=${DENOM}
+${CLI} erc20 transfer --rpc-address=${RPC_ADDRESS} --mnemonic="${MNEMONIC}" --from-index=${from_index} --to-address=${to_address} --amount=${amount} --denom=${DENOM}
```

Comment:
I noticed that the README has fixed tag which is v0.0.3, but it is possible to forget to update. 
How about changing it as follows?

```
$ git clone https://github.com/datachainlab/ethereum-ics20-cli.git
$ cd ethereum-ics20-cli
$ go install
CLI="ethereum-ics20-cli"
```

Reply:
Indeed! Specifying a branch name is not a good idea. Removed
https://github.com/datachainlab/ethereum-ics20-cli/pull/8/commits/5bc770060368621074543255e01693c7825d3370